lib/oli_web/live/common/cards_view_filter.ex

```diff
@@ -0,0 +1,46 @@
+defmodule OliWeb.Common.CardsViewFilter do
+  use Surface.Component
+
+  alias Surface.Components.Form.{Field, RadioButton}
+
+  slot default, required: true
+  prop table_model, :struct, required: true
+
+  def render(assigns) do
+    ~F"""
+    <div class="card mb-3">
+      <h3 class="card-header">Select Curriculum</h3>
+      <div class="card-body">
+        <p class="mt-1 mb-4">Select a curriculum source to create your course section.</p>
+
```

Comment:
Is it necessary to include this block in the component? It seems like a generic component that can be reused, and this text looks specific to one page.

Reply:
We can move that to a component prop once we use it somewhere else, but I agree maybe is clear to add that prop now, so will do! Thanks 😄